lib/Connection.php

```diff
@@ -43,6 +43,14 @@
 	 */
 	public $last_query;
 	/**
+	 * The total cumulative execution time (in millis) for queries.
+	 *
```

Comment:
Typically separate summary from description with a newline. Was trying to follow existing project style: https://github.com/robinpowered/php-activerecord/blob/master/lib/Reflections.php#L38-L45